simulation-debugging.md

```diff
@@ -58,11 +49,10 @@ make posix_sitl_lpe gazebo___gdb
 make posix_sitl_lpe gazebo___lldb
 ```
 
+where the last parameter is the &lt;viewer\_model\_debugger&gt; triplet (using three underscores implies the default &#39;iris&#39; model).
```

Comment:
@bkueng I think that the terminology is a bit confusing because the use of "viewer" rather than "simulator". Thoughts?

More generally, the [current simulator docs](https://dev.px4.io/en/simulation/#startingbuilding-sitl-simulation) don't mention the triplet or the debugger option (they refer to this pattern `[SIMULATOR]_[INIT_FILE]`). I'd like to add those to the linked doc, and then link to it from here.

Can you confirm:
1. Is the format of the last parameter more accurately portrayed as `[SIMULATOR][_INIT_FILE][_DEBUGGER]`?
2. Are there other options now than gdb and lldb?
3. Is the `___` just a shorthand for default when you want to specify the debugger too?

Reply:
The logic is defined here: https://github.com/PX4/Firmware/blob/master/src/firmware/posix/sitl_target.cmake#L103. So yes the `___` is a special case and will select `iris`. Options for debugger are: `none ide gdb lldb ddd valgrind callgrind`

Reply:
Thanks. Adding the information in https://github.com/PX4/Devguide/pull/386